ktor-intellij-plugin/resources/META-INF/plugin.xml

```diff
@@ -18,6 +18,12 @@
 
             <h2>Changelog</h2>
 
+            <h3>1.4.3</h3>
```

Comment:
I think the actual contents of the change log is missing?

Reply:
there were no changes in the generator itself. only new version for Ktor